### ISSUE TITLE
CVRF v1.2 prose section 5.1.2 missing `ProductTree` `Relationship`?

### ISSUE THREAD

**Issue by sthagen** [oasis-tcs/csaf#103]
As noted by @tschmidtb51 it seems CVRF v1.2 prose is missing the `ProductTree` `Relationship` in section [5.1.2](http://docs.oasis-open.org/csaf/csaf-cvrf/v1.2/cs01/csaf-cvrf-v1.2-cs01.html#_Toc493508905):
```
5.1.2 Product Tree – Full Product Name
Element prod:FullProductName
« The prod:FullProductName element MUST be a child of cardinality [1, ∞] for all possible locations inside the product tree representation. » [CSAF-5.1.2-1]
This elements instances have multiple possible parent elements: prod:ProductTree, prod:Releationship, and prod:Branch.
The prod:FullProductName elements define the endpoints of the Product Tree and occur either directly at the root level, at the branch level, or as the result of a relationship between two products.
The value of a Full Product Name element should be the product’s full canonical name, including version number and other attributes, as it would be used in a human-friendly document.
Attribute ProductID
The ProductID attribute is a token required to identify a Full Product Name so that it can be referred to from other parts in the document.
There is no predefined or required format for the ProductID as long as it uniquely identifies a product in the context of the current document.
Attribute CPE
The (Common Platform Enumeration) CPE attribute refers to a method for naming platforms external to CSAF CVRF.
« The CPE attribute if present MUST have a value, that is a valid cpe-lang:namePattern as defined in the external specification [CPE23_N] and related schemas. » [CSAF-5.1.2-2]
Non-normative comment:
Examples for ProductID values include incremental integers or Globally Unique Identifiers (GUIDs).
The structure for CPE as required for a valid CSAF CVRF document is described in its specification documents ([CPE23_N] et al.).
In short: The CPE can be either an integer (if there exists a known entry for the platform in question) or a candidate string from the vendor if no commonly registered entry yet exists (at the NIST CPE registry site).
Example 40:

<FullProductName ProductID="CVRFPID-0004">
  Microsoft Host Integration Server 2006 Service Pack 1
</FullProductName>
Example 41:

<FullProductName ProductID="CVRFPID-0005">
  Microsoft Office 2008 for Mac 12.3.1 Update
</FullProductName>
```
